content/post/personas.md

```diff
@@ -0,0 +1,49 @@
+| <img class="image" src="/images/persona2.svg" alt="Researcher persona" title="Researcher persona" ><br/>**Gordana**<br/>*Post-doctoral researcher (University of Fribourg - with dedicated SONAR portal)* | **Shared portal**<ul><li>display publicly a list of all my Swiss OA publications so that I can show my scientific accomplishments</li><li>...and all tasks of Fabrice</li></ul>**Dedicated portal**<ul><li>display publicly a list of all my publications (OA or not) related to my institution so that I can show my scientific accomplishments</li><li>upload full-texts of my pre-prints and post-prints so that I comply with the OA policy of my institution</li><li>enter and edit metadata about my publications so that they are searchable</li><li>export a list of all my publications (OA or not) related to my institution or to a specific project so that I can reuse the data</li><li>link my profile to standard identifiers (ORCID) so that my publications metadata are more accurate</li><li>import metadata from other platforms (ORCID, Google Scholar, etc.) so that I don't have to enter it manually</li><li>edit my personal profile</li><li>...and all tasks of Fabrice</li></ul> |
+| <img class="image" src="/images/persona3.svg" alt="Librarian persona" title="Librarian persona" ><br/>**Gabriele**<br/>*Librarian (Università della Svizzera italiana - with dedicated SONAR portal)* | **Shared portal**<ul><li>export metadata of publications according to various criteria (author, project, institution, document type, etc.) so that I can reuse the data (showcase, report, FNS, etc.)</li></ul>**Dedicated portal**<ul><li>upload PDF of pre-prints and post-prints (OA or not) of my institution's researchers so that he complies with the OA policy</li><li>indicate for each full-text the OA status (incl. embargo periods) so that they are publicly available or not.</li><li>enter metadata about publications (typology, rights, access, etc.) so that they are searchable</li><li>create profiles for researchers with specific rights</li><li>import metadata from other platforms (ORCID, Google Scholar, etc.) so that I don't have to enter it manually</li><li>validate full-text inputs of the researchers so that they complies with legal restrictions</li><li>validate and ev. complete metadata entered by researchers so that they complies with standards</li><li>export metadata of publications according to various criteria (author, project, institution, document type, …) so that I can reuse the data (showcase, report, etc.)</li><li>request or harvest metadata of publications according to various criteria (author, project, institution, document type, …) so that I can integrate them into other platforms (website, etc.)</li><li>ensure data integrity, readability and access so that we can access it in the long term</li></ul> |
```

Comment:
upload PDF of pre-prints and post-prints (OA or not) of my institution's researchers so that he complies with the OA policy --> so that they comply with

Reply:
import metadata from other platforms (ORCID, Google Scholar, etc.) so that I don't have to enter it manually --> correct d'utiliser "it" d'après moi. Harmoniser dans l'article (they, them --> it / are --> is)